### PR TITLE
[INLONG-848][Release] Add blog for the 1.9.0 release

### LINKED ISSUE #848: [Release] Add blog for the 1.9.0 release
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by luchunliang**
### Prepare a Pull Request

- Fixes #848 

### Motivation

Add blog for the 1.9.0 release

### CODE REVIEW

File: blog/2023-09-25-release-1.9.0.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+<!--truncate-->
+
+## About Apache InLong
+
+As the industry's first one-stop, full-scenario, open-source massive data integration framework, Apache InLong provides automatic, safe, reliable, and high-performance data transmission capabilities to facilitate businesses to build stream-based data analysis, modeling, and applications quickly. At present, InLong is widely used in various industries such as advertising, payment, social networking, games, artificial intelligence, etc., serving thousands of businesses, among which the scale of high-performance scene data exceeds 1 trillion lines per day, and the scale of high-reliability scene data exceeds 10 trillion lines per day.
+
+The core keywords of InLong project positioning are "one-stop" and "massive data". For "one-stop", we hope to shield technical details, provide complete data integration and support services, and implement out-of-the-box; With its advantages, such as multi-cluster management, it can stably support larger-scale data volumes based on trillions of lines per day.
+
+## Overview of version 1.8.0
```
Comment (vernedeng) [inline blog/2023-09-25-release-1.9.0.md:18]: 1.9.0

File: i18n/zh-CN/docusaurus-plugin-content-blog/2023-09-25-release-1.9.0.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+Apache InLong（应龙） 最近发布了 1.9.0 版本，该版本关闭了约 200+ 个issue，包含 2+ 个大特性和 30+ 个优化，主要完成了可观测性能力建设、优化DataProxySDK-CPP等。1.9.0 发布后，Apache InLong 在全链路跟踪、指标采集、接入及可视化观测、告警方面补齐了可观测能力建设，解决在开发和运营过程中的快速排查问题、性能优化等需求，优化Apache InLong运营运维的使用体验。
+<!--truncate-->
+
+## 关于 Apache InLong
+
+作为业界首个一站式、全场景海量数据集成框架，Apache InLong（应龙） 提供了自动、安全、可靠和高性能的数据传输能力，方便业务快速构建基于流式的数据分析、建模和应用。目前 InLong 正广泛应用于广告、支付、社交、游戏、人工智能等各个行业领域，服务上千个业务，其中高性能场景数据规模超百万亿条/天，高可靠场景数据规模超十万亿条/天。
+
+InLong 项目定位的核心关键词是“一站式”、“全场景”和“海量数据”。对于“一站式”，我们希望屏蔽技术细节、提供完整数据集成及配套服务，实现开箱即用；对于“全场景”，我们希望提供全方位的解决方案，覆盖大数据领域常见的数据集成场景；对于“海量数据”，我们希望通过架构上的数据链路分层、全组件可扩展、自带多集群管理等优势，在百万亿条/天的基础上，稳定支持更大规模的数据量。
+
+## 1.8.0 版本总览
```
Comment (vernedeng) [inline i18n/zh-CN/docusaurus-plugin-content-blog/2023-09-25-release-1.9.0.md:18]: 1.9.0

File: blog/2023-09-25-release-1.9.0.md (hunk 1 above)
Comment (luchunliang) [reply blog/2023-09-25-release-1.9.0.md]: Fixed

File: i18n/zh-CN/docusaurus-plugin-content-blog/2023-09-25-release-1.9.0.md (hunk 2 above)
Comment (luchunliang) [reply i18n/zh-CN/docusaurus-plugin-content-blog/2023-09-25-release-1.9.0.md]: Fixed

**Review by dockerzhang — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

**Pull request merged**